src/Tests/RulesDrupalTestBase.php

```diff
@@ -68,4 +82,16 @@ protected function createCondition($id) {
     return $condition;
   }
 
+  /**
+   * Helper callback to check if message is in the log.
```

Comment:
Method descriptions should be like "what does this do?" "Checks if a message is in the log."